docs/lib/javalib.rst

```diff
@@ -623,37 +623,33 @@ Embedding Resources
 -------------------
 
 In Scala Native, resources are implemented via embedding a resource in a resulting
-binary file. Only `getClass().getResourceAsInputStream()` is implemented.
-For that to work, you have to specify an additional NativeConfig option:
+binary file. Only ``getClass().getResourceAsInputStream()`` is implemented.
+For that to work, you have to specify an additional NativeConfig option::
 
-```scala
```

Comment:
I think this was suppose to be a code block.
https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#code-block-directive

Reply:
This is an example in `sbt.rst`
```
.. code-block:: scala

    import scala.scalanative.build._

    nativeConfig ~= {
      _.withLTO(LTO.thin)
        .withMode(Mode.releaseFast)
        .withGC(GC.commix)
    }

```